### PR TITLE
How to install ansible - my first article

### PR DESCRIPTION
**Pull request by ShowMeYourSkil**
My first articel about ansible. I hope you can push this on your site ;=)

### CODE REVIEW

File: content/posts/ansible-install.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+author: "ShowMeYourSkil"
+---
+
+# Ansible
+
+Hey, in this articel I would like to tell you something about Ansible and how you can install it. 
+
+# What is ansible?
+
+Ansible is a automation platform to automate process. Example like install full automatic Virtual Machines oder Windows updates and more..
```
Comment (mtorials) [inline content/posts/ansible-install.md:16]: processes. For example like installing

File: content/posts/ansible-install.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+It work with the language yml. You can create "Playbooks" through which an order is executed.
```
Comment (mtorials) [inline content/posts/ansible-install.md]: it works; so called playbooks

**Review by mtorials — COMMENTED** [review]
maybe some pcitures?

**Comment by ShowMeYourSkil** [comment]
> maybe some pcitures?

No. There is not much to see. When the playbooks arrive, I would like to include photos.

File: content/posts/ansible-install.md (hunk 2 above)
Comment (ShowMeYourSkil) [reply content/posts/ansible-install.md]: ?

File: content/posts/ansible-install.md (hunk 1 above)
Comment (ShowMeYourSkil) [reply content/posts/ansible-install.md:16]: I want add processes in my next articel... this articel is only how about install ansbile

Comment (mtorials) [reply content/posts/ansible-install.md:16]: i tried to mark some spelling and grammar mistakes.....

File: content/posts/ansible-install.md (hunk 2 above)
Comment (mtorials) [reply content/posts/ansible-install.md]: it workS <- you forgot an s

**Comment by mtorials** [comment]
could you fix the language mistakes? I marked them in the code review.

File: content/posts/ansible-install.md (hunk 1 above)
Comment (ShowMeYourSkil) [reply content/posts/ansible-install.md:16]: uhm no xD

I've give you write perms

File: content/posts/ansible-install.md (hunk 2 above)
Comment (ShowMeYourSkil) [reply content/posts/ansible-install.md]: oh thanks xD

**Review by ShowMeYourSkil — COMMENTED** [review]
I've finished my grammar correction

**Comment by ShowMeYourSkil** [comment]
> could you fix the language mistakes? I marked them in the code review.

I habe fix my mistakes. Sorry. please read my new commit in 5minuts

File: content/posts/ansible-install.md (hunk 1 above)
Comment (ShowMeYourSkil) [reply content/posts/ansible-install.md:16]: Please view my last commit. Bzw view this file now :D

**Comment by mtorials** [comment]
There are still some language mistakes. I can merge it and than correct it if you are ok with this.

**Comment by ShowMeYourSkil** [comment]
> There are still some language mistakes. I can merge it and than correct it if you are ok with this.

Ok

**Pull request merged**